### PR TITLE
Suggesting improvements for html tables in email

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
Suggesting improvements in the html tables look and feel.

### Testing
Tested in local environment as it is a standalone piece of code snippet.
Plain HTML for this change: http://pastebin.test.redhat.com/1103319

#### Before this Change
Screenshot from 2023-06-22 02-55-09

#### After this Change
after_change

### CODE REVIEW

File: cloud_governance/common/mails/mail_message.py [hunk 1]
```diff
@@ -391,24 +391,46 @@ def get_data_in_html_table_format(self, resources: dict):
...
+                    #customers th {
+                    padding-top: 12px;
+                    padding-bottom: 12px;
+                    text-align: left;
+                    background-color: #04AA6D;
+                    color: white;
+                    }
+                    </style>
+                """
+        html_table_format = style + '<table id="customers">'
```
Comment (ebattat) [inline cloud_governance/common/mails/mail_message.py:421]: Can you use format instead of +?
html_table_format =  f'{style} <table id="customers">'

**Review by ebattat — APPROVED** [review]

File: cloud_governance/common/mails/mail_message.py (hunk 1 above)
Comment (vishnuchalla) [reply cloud_governance/common/mails/mail_message.py]: Acknowledged. Changed and verified as suggested. Thank you!

**Review by athiruma — APPROVED** [review]
LGTM

**Review by ebattat — COMMENTED** [review]
Try to avoid using + for concatenating 2 strings because its allocate a fresh space in the memory.

File: cloud_governance/common/mails/mail_message.py [hunk 2]
```diff
@@ -391,24 +391,46 @@ def get_data_in_html_table_format(self, resources: dict):
...
             for resource in resource_data:
-                td_data = [resource.get(th_value) for th_value in thead_values]
-                if 'Deleted' == resource.get('Action'):
-                    html_table_format += f"""{tr_left }{''.join([f'{td_left_color}{value}{td_right}' for value in td_data])}{tr_right}"""
-                else:
-                    html_table_format += f"""{tr_left}{''.join([f'{td_left}{value}{td_right}' for value in td_data])}{tr_right}"""
+                html_table_format += '<tr>'
+                for th_value in thead_values:
+                    if 'Deleted' == resource.get(th_value):
+                        html_table_format += f"""<td>{resource.get(th_value)+" &#128465;"}</td>"""
```
[the remark this answers is not in the document]
Comment (ebattat) [reply cloud_governance/common/mails/mail_message.py:430]: @vishnuchalla did u try it ?

**Review by krishvoor — APPROVED** [review]
lgtm

**Pull request merged**